webapp/pages/admin/users.vue

```diff
@@ -33,6 +33,11 @@
             <b>{{ scope.row.name | truncate(20) }}</b>
           </nuxt-link>
         </template>
+        <template slot="email" slot-scope="scope">
+          <a :href="`mailto:${scope.row.email}`">
+            <b>{{ scope.row.email }}</b>
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-07T09:00:20Z" title="Monday, October 7th 2019, 11:00:20 am +02:00">Oct 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -33,6 +33,11 @@
             <b>{{ scope.row.name | truncate(20) }}</b>
           </nuxt-link>
         </template>
+        <template slot="email" slot-scope="scope">
+          <a :href="`mailto:${scope.row.email}`">
+            <b>{{ scope.row.email }}</b>
```


Some of those emails are quite long. Maybe we should think about how to fit the data into the layout? E.g. at some places in the app, we use `| truncate(20)` filter.

Although I'm OK to merge this and wait for our admins to complain about it :wink:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-07T09:38:39Z" title="Monday, October 7th 2019, 11:38:39 am +02:00">Oct 7, 2019</time>_

---


That's what I thought as well, @datenbrei mentioned that they might be copying and pasting the links, so maybe it's more difficult if it's truncated??

Reply:
> [<img alt="aonomike" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/aonomike) **Authored by [aonomike](https://github.com/aonomike)**
_<time datetime="2019-10-07T10:34:45Z" title="Monday, October 7th 2019, 12:34:45 pm +02:00">Oct 7, 2019</time>_

---


Thanks @mattwr18 @roschaefer 
Let me know if I need to update this :D